### ISSUE TITLE
interfaces implementing interfaces

### ISSUE THREAD

**Issue by fotoetienne** [gabotechs/graphqxl#48]
In the current release of the GraphQL spec, interfaces can [implement interfaces](https://github.com/graphql/graphql-spec/pull/373)

```graphql
interface Node {
  id: ID!
}

interface Resource implements Node {
  id: ID!
  url: String
}
```

This would be great to support in GraphQXL

**Comment by gabotechs** [comment]
Agree, this should be supported in GraphQXL, I will prepare a PR for this soon

**Comment by taras** [comment]
It looks like it's been implemented. Does the playground have support for this?

**Comment by gabotechs** [comment]
yeah, it should, it was released in the current version v0.39.0, which is still not officially released in `brew`, but should be available in the playground and in the node wrapper. Was planning on closing this once it is officially released in brew

**Comment by gabotechs** [comment]
Apparently the official `homebrew-core` repository has a bot that already checks for updates, this functionality was automatically published in homebrew some time ago https://github.com/Homebrew/homebrew-core/pull/122392.

Closing it as it is now public everywhere
